src/main/java/com/matera/cadastrocentral/address/Address.java

```diff
@@ -0,0 +1,99 @@
+package com.matera.cadastrocentral.address;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.UUID;
+
+@Entity
+public class Address {
+
+    private @Id @GeneratedValue UUID address_id;
```

Comment:
can you please remove the @GeneratedValue  anotation? when we use UUIDs for entity identification we create it on application not on database

Reply:
Sorry, I thought I had to let the database generate it automatically. Should I add a UUID uuid = UUID.randomUUID() in the constructor?

Reply:
you can create it in the service or you can add a PrePersist annotaded method on the entity, like:

```
@PrePersist
public void prePersist() {
    this.addressId = UUID.randomUUID();
}
```